File: PruebaPacientes/AplicacionWeb/Models/FormularioParaAnalisis.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Modelo
{
    public class FormularioParaAnalisis
    {
        public int Id                                                              { get; set; }
        public EstadoDelFormulario EstadoDelFormulario                             { get; set; }
        public DateTime FechaDeCreacion                                            { get; set; }
        public Paciente Paciente { get; set; }
        public int PacienteId { get; set; }
        public AntecedenteEpidemiologicos1 AntecedenteEpidemiologicos1             { get; set; }
        public int AntecedenteEpidemiologicos1Id                                   { get; set; }
        public AntecedenteEpidemiologicos2 AntecedenteEpidemiologicos2             { get; set; }
        public int AntecedenteEpidemiologicos2Id                                   { get; set; }
        public EnfermedadesPreviasComorbilidades EnfermedadesPreviasComorbilidades { get; set; }
        public int EnfermedadesPreviasComorbilidadesId                             { get; set; }
        public SignosYSintomas SignosYSintomas                                     { get; set; }
        public int SignosYSintomasId                                               { get; set; }
        public string RespuestaMedica                                              { get; set; }
    }
}

File: Frontend/api/models/FormularioParaAnalisis.js
/**
 * FormularioParaAnalisis.js
 *
 * @description :: A model definition represents a database table/collection.
 * @docs        :: https://sailsjs.com/docs/concepts/models-and-orm/models
 */

module.exports = {

  attributes: {

    EstadoDelFormulario:{ //modelo Uno-Muchos
      model:'EstadoDelFormulario'
    },
    FechaDeCreacion:{
      type:'string'
    },
    /*Paciente:{ //modelo Uno-Uno
      collection:'Paciente',
      via:'owner'
    },*/
    /*PacienteId:{
      type:'number',
    },*/
    AntecedenteEpidemiologico1:{ //modelo Uno-Uno
      collection:'AntecedenteEpidemiologicos1',
      via:'owner'
    },
    /*AntecedenteEpidemiologico1Id:{ 
      type:'number',
    },*/
    AntecedenteEpidemiologico2:{ //modelo Uno-Uno
      collection:'AntecedenteEpidemiologicos2',
      via:'owner'
    },
    /*AntecedenteEpidemiologico2Id:{ 
      type:'number',
    },*/
    EnfermedadesPreviasComorbilidades:{ //Modelo Uno-Uno
      collection:'EnfermedadesPreviasComorbilidades',
      via:"owner"
    },
    /*EnfermedadesPreviasComorbilidadesId:{
      type:'number',
    },*/ 
    SignosYSintomas:{ //modelo Uno-Uno
      collection:'SignosYSintomas',
      via:"owner"
    },
    /*SignosYSintomasId:{
      type: 'number'
    },*/
    RespuestaMedica:{
      type:'string'
    },

  },

};

